resources/less/virkailija-application.less

```diff
@@ -1086,6 +1086,69 @@ and (max-width: 1000px) {
   margin-bottom: 50px;
 }
 
+.application-handling__application-version-history-container {
+  position: fixed;
+  background-color: rgba(53, 53, 53, 0.71);
```

Comment:
Näitä värejä (myös rgba) vois taas ekstraktoida muuttujiksi?

Reply:
:+1: